packages/shared/lib/wallet.ts

```diff
@@ -480,24 +480,13 @@ export async function asyncSyncAccountOffline(account: WalletAccount): Promise<v
     })
 }
 
-export const asyncGetNodeInfo = (accountId: string, url?: string, auth?: IAuth): Promise<INodeInfo> => {
+export const asyncGetNodeInfo = (url?: string, auth?: IAuth): Promise<IStardustNodeInfo> => {
     if (!url || (!url && !auth)) {
-        const node = get(activeProfile)?.settings?.networkConfig?.nodes.find((n) => n.isPrimary)
-
-        url = node?.url
-        auth = node?.auth
+        return new Promise((resolve) => resolve(<IStardustNodeInfo>{}))
     }
 
-    return new Promise<INodeInfo>((resolve, reject) => {
-        api.getNodeInfo(accountId, url, auth, {
-            onSuccess(response) {
-                resolve(response.payload)
-            },
-            onError(err) {
-                reject(err)
-            },
-        })
-    })
+    const manager = get(profileManager)
+    return manager.getNodeInfo(url, auth)
```

Comment:
async function but we are not awaiting on this?

Reply:
It's not a in a try / catch block so doesn't need the `await` keyword.